indico/web/client/js/jquery/extensions/global.js

```diff
@@ -247,19 +247,26 @@ $(document).ready(function() {
   showFormErrors();
 
   // Show form creation dialog if hash is present in the URL
-  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d+))?$/);
+  let match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d*)(?::([a-f0-9-]{36}))?)?$/);
   if (match) {
-    var eventType = match[1];
-    var categoryId = match[2];
-    var title = {
+    let eventType = match[1];
+    let categoryId = match[2];
+    let eventUUID = match[3];
+    let title = {
```

Comment:
I don't think we ever reassign those, so `const` it is.

For the match you could also do something like this:

```js
const [, eventType, categoryId, eventUUID] = match;
```

Reply:
Much cleaner, thanks.